src/helpers/checkDateHelper.js

```diff
@@ -11,4 +11,14 @@ const checkDate = (res, startDate, endDate) => {
     responseUtil(res, 400, strings.accommodation.error.DATE_ERROR);
   }
 };
-export default checkDate;
+
+const statsDate = (res, startDate, endDate) => {
+
+  if (moment(startDate) < moment(endDate)) {
```

Comment:
Don`t we already have a function for comparing dates?
Also, I think this condition should be the other way round since `startDate` should be lesser than `endDate`